torch/ao/quantization/pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -369,6 +472,124 @@ def _annotate_conv2d(
                 continue
             self._annotate_conv_node_helper(conv_node, True, quantization_config)
 
+    def _annotate_maxpool2d(
+        self, node: Node, quantization_config: QuantizationConfig
+    ) -> None:
+        if (
+            node.target is not torch.ops.aten.max_pool2d_with_indices.default
+            or not (
+                len(list(node.users)) == 1
+                and (list(node.users)[0].target == operator.getitem)
+            )
+        ):
```

Comment:
If it is a pattern, why not handle it like conv fusion but treat it as a single op?

Reply:
Thanks for the comment. After second thought, I think `_annotate_quantizable_single_op` may not be a good name here which causes confusion. And I think we should not annotate `maxpool2d` like `conv fusion`. Here is some of my understanding:

- The difference between `conv` and `maxpool2d` is not a pattern or single op.
- `Conv` should be more like the `lower_precision_fp` op set in `Autocast`, which we will force to run with lower precision to get better performance.
- `Maxpool2d` is more like the fall-through op set in `Autocast`, which will run with lower precision if previous node does so.

So, how about we change the function name from `_annotate_quantizable_single_op` to something may be like `_annotation_propagation_quantizable_pattern`? @jgong5 do you have any suggestions here?

Reply:
Then, maybe we can abandon the concept of `single quantized node` to unify `Conv` and `Maxpool2d` as `quantized pattern`.